model/inline/src/main/scala/aqua/model/inline/ArrowInliner.scala

```diff
@@ -431,14 +473,15 @@ object ArrowInliner extends Logging {
   ): State[S, (OpModel.Tree, List[ValueModel])] =
     for {
       passArrows <- Arrows[S].pickArrows(call.arrowArgNames)
+      arrowsState <- Arrows[S].arrows
       arrowsFromAbilities <- call.abilityArgs
         .traverse(getAllArrowsFromAbility)
         .map(_.fold(Map.empty)(_ ++ _))
 
       inlineResult <- Arrows[S].scope(
         for {
           _ <- Arrows[S].resolved(passArrows ++ arrowsFromAbilities)
-          inlineResult <- ArrowInliner.inline(arrow, call)
+          inlineResult <- ArrowInliner.inline(arrow, call, arrowsState)
```

Comment:
I don't get it. On the line above `arrowsFromAbilities` are resolved. But `arrowState` is still passed and then used to retrieve abilities arrows. Are both things needed?